bigflow/configuration.py

```diff
@@ -10,12 +11,9 @@ def __init__(self,
         self.properties = properties
 
 
-class StringStream:
-    def __init__(self):
-        self.value = ''
-
-    def write(self, text):
-        self.value += text
+def current_env():
+    """Returns current env name (specified via 'bighflow --config' option)"""
+    return os.environ.get('env')
```

Comment:
I think it's `bf_env`:

```
def set_configuration_env(env):
    """
    Sets 'bf_env' env variable
    """
    if env is not None:
        os.environ['bf_env'] = env
        print(f"bf_env is : {os.environ.get('bf_env', None)}")
```

BTW. the `bf_` part is the default value for `environment_variables_prefix` in `Config`. The problem is that CLI always assumes that the `environment_variables_prefix` value is `bf_`.

Reply:
Fixed.